webanno-brat/src/main/java/de/tudarmstadt/ukp/clarin/webanno/brat/annotation/BratAnnotationEditor.java

```diff
@@ -103,11 +107,13 @@
     private static final String PARAM_OFFSETS = "offsets";
     private static final String PARAM_TARGET_SPAN_ID = "targetSpanId";
     private static final String PARAM_ORIGIN_SPAN_ID = "originSpanId";
+    private static final String PARAM_SPAN_TYPE = "type";
 
     private @SpringBean AnnotationSchemaService annotationService;
 
     private WebMarkupContainer vis;
     private AbstractAjaxBehavior requestHandler;
+    private OnClickActionParser onClickActionParser = new OnClickActionParser();
```

Comment:
The parser is stateless. I would suggest making it's method static and calling it directly.